.github/workflows/tests.yml

```
@@ -44,9 +44,17 @@ jobs:
         if: matrix.php >= 7.2
         run: composer require phpunit/phpunit:^5.7.27 --no-update --no-interaction --dev
 
+      - name: Require PHPStan
+        if: matrix.php == 7.4 && matrix.setup == 'stable'
+        run: composer require phpstan/phpstan:^0.12.68 --no-interaction --dev
```

```suggestion
        run: composer require phpstan/phpstan:^0.12.68 --no-interaction --dev --no-update
```
As the update is ran just after, it would be faster.

---

This didn't seam to work since the step is skipped if we had a cache hit on composer.lock earlier in the process.

---

I see but actually, we want this package to be cached too. So'll you'll have to change `restore-keys` and `keys` so it uses a clean cache storage. This will change anyway if we move it into a dedicated file.